### ISSUE TITLE
[SFSF][A11y][MultiComboBox] Focus is lost after deleting the tokens and notification message is being read out by screen readers

### ISSUE THREAD

**Issue by saorabhkr** [SAP/ui5-webcomponents-react#4278]
### Describe the bug

Describe the bug

With MultiComboBox, when we delete any tokens we have observed two things: 

> After deleting any token focus is not staying there on the previous/next token.

> Upon deletion screen reader is not announcing "Token deleted" or anything. 

The same can be seen here:
https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story

### Reproduction steps

Go to [Demo Page](https://sap.github.io/ui5-webcomponents-react/?path=/story/inputs-multicombobox--default-story)

Select a few tokens from the dropdown
Delete the tokens
It's been observed that it's not retaining the focus.
Now delete any token and the announcement by NVDA/JAWS
It's not announcing anything.

### Expected Behaviour

MultiComboBox:

> After deleting any token focus should stay on the previous/next token

> After deleting any token screen reader should announce some message eg: "Token deleted"
on

### UI5 Web Components for React Version

1.8.1

### UI5 Web Components Version

1.10.5

### Browser

Chrome

### Operating System

MacOS

**Comment by Lukas742** [comment]
Hi @saorabhkr 

Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

> After deleting any token focus is not staying there on the previous/next token.

I have not been able to reproduce this. Please provide detailed steps on how this behavior occurs.

https://user-images.githubusercontent.com/9749730/223658875-7a996173-dfed-4bea-b0e9-4de2ff504b53.mp4

__

Hi colleagues, 

there is no dedicated announcement for token deletion. Could you please check if this is intentional? 
There could also be an issue with focus after token deletion, but let's wait for the input of the author for a more detailed description of how to reproduce this.
